src/read.rs

```diff
@@ -0,0 +1,201 @@
+// Copyright 2021 by Accenture ESR
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! # dlt reading support
```

Comment:
License text only once in the LICENCE file, in each source file only a one-liner
```
// Copyright 20.. Accenture.
```

Reply:
Created an issue for this: https://github.com/esrlabs/dlt-core/issues/46